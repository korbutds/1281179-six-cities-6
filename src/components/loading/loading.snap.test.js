import React from 'react';
import {render} from '@testing-library/react';
import LoaderScreensaver from './loading';

test(`Should render LoaderScreensaver component correctly`, () => {
  const {container} = render(<LoaderScreensaver/>);
  expect(container).toMatchSnapshot();
});
