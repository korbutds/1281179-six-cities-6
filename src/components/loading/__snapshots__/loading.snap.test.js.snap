// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render LoaderScreensaver component correctly 1`] = `
<div>
  <div
    class="cities"
  >
    <div
      class="cities__places-container container"
    >
      <div
        class="sc-bdfBwQ iQWhUR"
      >
        <div
          aria-busy="true"
          class=""
        >
          <svg
            aria-label="audio-loading"
            height="100"
            stroke="#4481c3"
            viewBox="0 0 38 38"
            width="100"
            xmlns="http://www.w3.org/2000/svg"
          >
            <g
              fill="none"
              fill-rule="evenodd"
            >
              <g
                stroke-width="2"
                transform="translate(1 1)"
              >
                <circle
                  cx="18"
                  cy="18"
                  r="18"
                  stroke-opacity=".5"
                />
                <path
                  d="M36 18c0-9.94-8.06-18-18-18"
                >
                  <animatetransform
                    attributeName="transform"
                    dur="1s"
                    from="0 18 18"
                    repeatCount="indefinite"
                    to="360 18 18"
                    type="rotate"
                  />
                </path>
              </g>
            </g>
          </svg>
        </div>
      </div>
    </div>
  </div>
</div>
`;
